lib/src/value/calculation.dart

```diff
@@ -169,19 +587,20 @@ class SassCalculation extends Value {
   /// a [CalculationInterpolation].
   static Object operate(
           CalculationOperator operator, Object left, Object right) =>
-      operateInternal(operator, left, right, inMinMax: false, simplify: true);
+      operateInternal(operator, left, right,
+          inLegacySassFunction: false, simplify: true);
 
-  /// Like [operate], but with the internal-only [inMinMax] parameter.
+  /// Like [operate], but with the internal-only [inLegacySassFunction] parameter.
   ///
-  /// If [inMinMax] is `true`, this allows unitless numbers to be added and
+  /// If [inLegacySassFunction] is `true`, this allows unitless numbers to be added and
   /// subtracted with numbers with units, for backwards-compatibility with the
   /// old global `min()` and `max()` functions.
```

Comment:
If this parameter is being updated to cover other legacy, the documentation should as well.